core/templates/pages/error-pages/error-404/error-404-page-root.component.html

```diff
@@ -0,0 +1,7 @@
+<oppia-angular-root>
+  <oppia-base-content>
+    <content>
```

Comment:
Generall, I see these tags in *.mainpage.html file, is this going to be in the page compoenent file in the future?

Reply:
yeah, angular does not allow transclusion on root components unlike ajs. So, `oppia-angular-root` needed to be wrapped with a wrapper component.

Reply:
I see, thanks for the epxlantion!